src/webgpu/api/operation/command_buffer/copyTextureToTexture.spec.ts

```diff
@@ -1136,16 +1136,15 @@ g.test('copy_multisampled_color')
           [[group(0), binding(0)]] var sourceTexture : texture_multisampled_2d<f32>;
           [[group(0), binding(1)]] var destinationTexture : texture_multisampled_2d<f32>;
           [[stage(fragment)]]
-          fn main() -> [[location(0)]] vec4<f32> {
-            for (var sampleIndex = 0; sampleIndex < 4; sampleIndex = sampleIndex + 1) {
+          fn main([[builtin(position)]] coord_in: vec4<f32>) -> [[location(0)]] vec4<f32> {
+            var coord_in_vec2 = vec2<i32>(i32(coord_in.x), i32(coord_in.y));
```

Comment:
I think we need to scale by the render target size. `builtin(position)` is between -1 and 1 IIRC

Reply:
In the vertex shader, `[[builtin(position)]]` is like gl_Position (-1 to 1 in x/y).
In the fragment shader, `[[buildin(position)]]` is like gl_FragCoord (0 to width/height in pixels).

The confusing wording in the spec is a known issue, but I just filed https://github.com/gpuweb/gpuweb/issues/2106 to actually fix it.

Reply:
Thanks, I indeed got confused by the spec :)

Reply:
@Jiawei-Shao the comment I made was incorrect so please land after removing the last fixup you made.

Reply:
Oh I thought @Kangz's comment after I read the below statements in the SPEC. So it is incorrect, right?
```
Framebuffer position of the current fragment, using **normalized homogeneous coordinates**.
```

Reply:
The spec is probably incorrect. You could look at the generated HLSL: if it is using SV_Position directly then it means that it is in pixels and not homogenous coordinates.

Reply:
OK. Then we need to fix the SPEC as it is really confusing right now.